packages/js/product-editor/src/components/tab/block.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-tab",
+	"title": "Product tab",
+	"category": "woocommerce",
+	"description": "The product tab.",
+	"keywords": [ "products", "tab", "group" ],
+	"textdomain": "default",
+	"attributes": {
+		"id": {
```

Comment:
Would it be useful to have support for `disabled` via an attribute here?

Reply:
Definitely.  Planned for the follow-ups to tabs, but I'd like for us to flesh out how we plan on "templating" different product types.

If we go with individual templates per product type, attributes may work well, but otherwise we may have to build logic directly into the block.